<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form1</class>
 <widget class="QWidget" name="Form1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>657</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="madeby_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>510</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font-weight:bold;
font-size:25px;
color:rgb(74, 74, 74);</string>
   </property>
   <property name="text">
    <string>Made By Tharun R</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resource_Final.qrc">:/First/Final_File/Main_Logo.png</pixmap>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_url_search">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>661</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0;background:transparent</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="search_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(74, 74, 74);border-radius:20px;background:transparent
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource_Final.qrc">:/First/Final_File/Search_Logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(74, 74, 74);
border-radius:19px;
font-size:15px;
padding-left:50px;
color:white;</string>
    </property>
   </widget>
   <zorder>lineEdit</zorder>
   <zorder>search_label</zorder>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font-weight:bold;
font-size:20px;
color:rgb(74, 74, 74);</string>
   </property>
   <property name="text">
    <string>Ex: https://www.youtube.com/watch?v=dQw4w9WgXcQ</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="Back_Label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(235, 5, 182, 255), stop:1 rgba(255, 158, 251, 255))</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Back_Label</zorder>
  <zorder>madeby_label</zorder>
  <zorder>group_url_search</zorder>
  <zorder>label</zorder>
  <zorder>logo_label</zorder>
 </widget>
 <resources>
  <include location="Resource_Final.qrc"/>
 </resources>
 <connections/>
</ui>
